webapp/src/Twig/TwigExtension.php

```diff
@@ -159,6 +159,7 @@ public function getGlobals(): array
             'doc_links'                     => $this->dj->getDocLinks(),
             'allow_registration'            => $selfRegistrationCategoriesCount !== 0,
             'enable_ranking'                => $this->config->get('enable_ranking'),
+            'new_version_available'         => $this->dj->checkNewVersion(),
```

Comment:
From what I see and tried, this will make a HTTP request to `versions.domjudge.org` on every page load. This seems very aggressive, and might even cause an accidental DDOS of our server when large contests are running. Should we store it with some cache time in the DB?